emscripten/buildToolkit

```diff
@@ -246,6 +246,7 @@ system($ccCmd);
 if ($? == 0) {
 	print " Done.\n";
 
+    `npm --prefix npm install`;
```

Comment:
Isn't this adding a `npm/node_modules` directory? and a `npm/package-lock.json` file? This is what I can now see after having run `buildNpmPackage`. I am not sure what will happen with these when I do an `npm publish` from there. Do you know if they will be published as well? I don't think we want that, do we?

Reply:
Good. Then they should be added to `.gitignore`

Reply:
They should be ignored in `emscripten/npm/.gitignore`. Also `emscripten/npm/dist` should not be commited and is ignored in the same file.